source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/list/__tests__/filter-by-organization-acceptance-test.tsx

```diff
@@ -0,0 +1,134 @@
+        attributes: {
+          name: 'DeveloperTown'
+        }
+      }, {
+        type: 'organizations', id: 3,
+        attributes: {
+          name: 'Kalaam Media'
+        }
+      }]
+    });
+
+    beforeEach(function() {
+      this.mockGet(200, '/groups', {
+        data: [{
+          type: 'groups',
+          id: '2',
+          attributes: {
+            name: 'Fake group'
+          }
+        }]
+      });
+    });
+
+    beforeEach(function () {
+      this.mockGet(200, '/users', {
+        data: [{
+          type: 'users',
+          id: 2,
+          attributes: { id: 1, familyName: 'fake', givenName: 'One' },
+          relationships: {
+            ['organization-memberships']: { data: [ { id: 2, type: 'organization-memberships' }]},
+            "group-memberships": {}
+          }
+        }, {
+          type: 'users',
+          id: 3,
+          attributes: { id: 1, familyName: 'fake', givenName: 'Two' },
+          relationships: {
+            ['organization-memberships']: { data: [ { id: 3, type: 'organization-memberships' }]},
+            "group-memberships": {}
+          },
+        }],
+        included: [
+          {
+            id: 2, type: 'organization-memberships',
+            attributes: {},
+            relationships: {
+              user: { data: { id: 2, type: 'users' } },
+              organization: { data: { id: 1, type: 'organizations' } }
+            }
+          },
+          {
+            id: 3, type: 'organization-memberships',
+            attributes: {},
+            relationships: {
+              user: { data: { id: 3, type: 'users' } },
+              organization: { data: { id: 2, type: 'organizations' } }
+            }
+          }
+        ]
+      });
+    });
+
+    describe('Select all organizations',() => {
+
+      beforeEach(async function () {
+        await visit('/users');
+        await app.openSidebar();
+        await app.openOrgSwitcher();
+        await switcher.selectAllOrg();
+      });
+
+      describe('Renders users page', () => {
+
+        it('Should see all users', () => {
+
+          expect(page.usernames().length).to.equal(3);
+
+          const usernames = page.usernames();
+          const text = usernames.map(u => u.text).join();
+
+          expect(text).to.include('fake fake');
+          expect(text).to.include('One fake');
+          expect(text).to.include('Two fake');
+        });
+
+        describe('Select a specific organization', () => {
+
+          beforeEach(async function () {
+            await app.openSidebar();
+            await app.openOrgSwitcher();
+            await switcher.selectOrg();
+          });
+
+          it('Only the user that belongs to the same organization are displayed', () => {
+
+            expect(page.usernames().length).to.equal(2);
+
+            const usernames = page.usernames();
+            const text = usernames.map(u => u.text).join();
+
+            expect(text).to.include('fake fake');
```

Comment:
there should be additional assertions here telling me that the other two users aren't here, cause as-is, I don't have enough information to make that assumption.